packages/mobile/src/components/now-playing-drawer/NowPlayingDrawer.tsx

```diff
@@ -118,6 +119,13 @@ export const NowPlayingDrawer = memo(function NowPlayingDrawer(
 
   const { drawerNavigation } = useContext(AppDrawerContext)
 
+  // Don't show shadow when we're in chat screen, so that the drawer appears
+  // at same level as the chat screen textinput
+  const isInChatScreen = useNavigationState((state) => {
```

Comment:
nice! yeah i think its def hacky, but at least consistent with the other hack we have for this type of introspection